client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -1925,9 +1925,15 @@ def finetune(
                 # launch fine-tuning
                 data = {
                     "base_model_version_id": self.id,
+                    "destination_model_version_id": 0,
```

Comment:
out of curiosity, why 0? shouldn't it be provided by the user?

Reply:
🤦